sdk/go/common/util/env/env.go

```diff
@@ -287,7 +285,7 @@ func (b BoolValue) Value() bool {
 	if !ok {
 		return false
 	}
-	return cmdutil.IsTruthy(v)
+	return v == "1" || strings.EqualFold(v, "true")
```

Comment:
Needed to inline this to prevent a module import cycle with cmdutil and tokens and env